packages/@react-aria/dnd/docs/dnd.mdx

```diff
@@ -0,0 +1,90 @@
+import OnDropPosition from './OnDropPosition.svg';
+import RootDropPosition from './RootDropPosition.svg';
+import DragPreview from './DragPreview.svg';
+import DragAffordance from './DragAffordance.svg';
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+---
+category: Drag and Drop
+keywords: [drag, drop, dnd, drag and drop, aria, accessibility]
+navigationTitle: Introduction
+---
+
+# Drag and Drop
+
+Drag and drop is an intuitive way for users to transfer data between locations. React Aria implements drag and drop for mouse and touch interactions, and provides full keyboard and screen reader accessibility.
+
+## Introduction
+
+Drag and drop is a common UI interaction that allows users to transfer data between two locations by directly moving a visual representation on screen.
+It is a flexible, efficient, and intuitive way for users to perform a variety of tasks, and is widely supported across both desktop and mobile operating systems.
+In addition to the standard mouse and touch interactions, React Aria also implements keyboard and screen reader accessible alternatives for drag and drop to enable all users to perform these tasks.
+
+## Concepts
+
+Drag and drop allows a user to move data between two locations. The initial location is referred to as a **drag source**, and the final location is referred to as a **drop target**. The source and target may be within the same application, for example two adjacent lists, or in different applications, for example uploading files from the desktop.
+
+The drag source is responsible for providing one or more **drag items**, which specify the data to be dragged. A drag item may be an object represented by the draggable element, a file, plain text content, etc. Each drag item includes a type, and the actual data for the item. The type of a drag item can be one of the [common mime types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types) or a custom type specific to the application. Multiple representations of the same object may be provided in different formats for interoperability with various drop targets. This allows drag and drop to work even between different applications. For example, a drag source may provide the data for a drag in both a custom object format and as plain text. This allows dropping within the application to include richer functionality, but also allows the user to drop in external applications such as email clients or text editors.
+
+While dragging, a **drag preview** is displayed under the user's mouse or finger to indicate the items being dragged. By default, this is a copy of the dragged element, but it can be customized. For example, when multiple items are dragged, they could be shown as a stack, or with a badge indicating the count.
+
+<figure style={{display: 'flex', flexDirection: 'column', alignItems: 'center', margin: '20px 0'}}>
+  <DragPreview role="img" aria-labelledby="drag-preview-caption" />
```

Comment:
We could potentially also use these visuals on the react spectrum version of this page too. Might help break up the wall of text a bit.

Reply:
definitely, I'll add a task item in the board to do so